src/components/curriculum/CurriculumSection.js

```diff
@@ -73,6 +74,8 @@ const CurriculumSection = props => {
     toggleNavigateTo,
     showLinkToCurriculum = true,
   } = props
+  // TODO: Remove traingType once all upcoming workshops have type REACT_WORKSHOP or GRAPHQL_WORKSHOP as it should only be a string
+  const trainingType = type.length < 5 ? type[0] : type
```

Comment:
This is some more tempory logic to handle the issue we currently have of needing to handle multiple workshop types coming from the API, it should be refactored back as soon as the dates have passed and we deprecate those types. At that point they should just be strings as they were previously.